packit_service/worker/jobs.py

```diff
@@ -259,17 +266,15 @@ def report_task_accepted(
             return
 
         job_helper = self.initialize_job_helper(handler_kls, job_config)
-        reporting_method = None
 
         if isinstance(job_helper, ProposeDownstreamJobHelper):
             reporting_method = job_helper.report_status_to_all
 
-        elif isinstance(job_helper, BaseBuildJobHelper):
-            reporting_method = (
-                job_helper.report_status_to_tests
-                if handler_kls == TestingFarmHandler
-                else job_helper.report_status_to_build
-            )
+        elif isinstance(job_helper, TestingFarmJobHelper):
+            reporting_method = job_helper.report_status_to_tests
+
+        else:
+            reporting_method = job_helper.report_status_to_build
```

Comment:
Can't we add a new alias method on the helper level to hide this detail? (Something like `report_status_to_current_job`/`report_status_to_configured_job`/...)

Reply:
great idea, added, PTAL